### PR TITLE
Add section on decisions and processing-pull-requests

### PR DESCRIPTION
**Pull request by csarven**
Further details decisions and processing PRs based on https://www.w3.org/Consortium/Process/#correction-classes

### CODE REVIEW

**Review by elf-pavlik — APPROVED** [review]

**Review by TallTed — CHANGES_REQUESTED** [review]
Language tweaks for clarity

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -346,6 +346,23 @@ We also make the following recommendations:
...
+|[No changes to text content](https://www.w3.org/2023/Process-20230612/#class-1)|Editors MAY PR and merged at their discretion.|Within 5 days or 2 meetings.|
+|[Changes that do not functionally affect interpretation of the document](https://www.w3.org/2023/Process-20230612/#class-2)|Editors SHOULD PR and merged at their discretion.|Within 5 days or 2 meetings.|
+|[Other changes that do not add new features](https://www.w3.org/2023/Process-20230612/#class-3)|MUST PR.|Within 10 days or 2 meetings.|
+|[New features](https://www.w3.org/2023/Process-20230612/#class-4)|MUST PR.|Within 10 days or 2 meetings.|
+|[Changes to the contents of a registry table](https://www.w3.org/2023/Process-20230612/#class-5)|Editors or chairs MUST PR.|Within 10 days or 2 meetings.|
+
+
 ## Vocabulary Management
 
 TRs might refer or make use of namespaces for specific functionality. The CG
```
[the remark this answers is not in the document]
Comment (csarven) [reply CONTRIBUTING.md:368]: Got this in https://github.com/solid/specification/commit/3c0b609dea054ca8698ce8d40637412584d9f2b5 , thanks!

**Review by kjetilk — APPROVED** [review]
LGTM

**Comment by csarven** [comment]
Thank you @TallTed @elf-pavlik @kjetilk for reviewing.

Merging as per agreement in [2023-11-01 CG meeting](https://github.com/solid/specification/blob/main/meetings/2023-11-01.md#add-section-on-decisions-and-processing-pull-requests).

**Pull request merged**